distro/configs/openmrs/frontend_assembly/spa-assemble-config.json

```diff
@@ -1,12 +1,41 @@
 {
   "frontendModules": {
-    "@openmrs/esm-implementer-tools-app": "latest",
+    "@openmrs/esm-cohort-builder-app": "latest",
+    "@openmrs/esm-home-app": "latest",
+    "@openmrs/esm-system-admin-app": "latest",
     "@openmrs/esm-devtools-app": "latest",
-    "@openmrs/esm-form-builder-app": "next",
-    "@openmrs/esm-form-engine-app": "next"
+    "@openmrs/esm-patient-flags-app": "latest",
+    "@openmrs/esm-openconceptlab-app": "latest",
+    "@openmrs/esm-patient-search-app": "latest",
+    "@openmrs/esm-primary-navigation-app": "latest",
+    "@openmrs/esm-active-visits-app": "latest",
+    "@openmrs/esm-patient-lists-app": "latest",
+    "@openmrs/esm-patient-list-management-app": "latest",
+    "@openmrs/esm-login-app": "latest",
+    "@openmrs/esm-patient-banner-app": "latest",
+    "@openmrs/esm-patient-registration-app": "latest",
+    "@openmrs/esm-dispensing-app": "latest",
```

Comment:
Looking at this now, there might be no need for these overrides if we've already added them in the `reference-application-spa-assemble-config.json`

Reply:
if we don't specify the [distro/configs/openmrs/frontend_assembly/spa-assemble-config.json](https://github.com/MSF-OCG/LIME-EMR/pull/10/files/15a8466f4fc8d5b144e13185a89248b840a697c2#diff-d4f1e82423e969952d8e7058151bc5e128ece3b24b902e638f8571abbc9e3928) file, the frontend will not build but will copy artifacts from ozone.  see more [here](https://github.com/ozone-his/ozone/blob/main/scripts/openmrs/frontend_assembly/build-openmrs-frontend.groovy#L20-L21)